source/agora/consensus/data/genesis/Coinnet.d

```diff
@@ -0,0 +1,92 @@
+/*******************************************************************************
+
+    Defines the genesis block used by the live network (CoinNet)
+
+    Copyright:
+        Copyright (c) 2019-202 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+module agora.consensus.data.genesis.Coinnet;
+
+import agora.common.Amount;
+import agora.common.Hash;
+import agora.common.Types;
+import agora.common.crypto.Key;
+import agora.consensus.data.Block;
+import agora.consensus.data.Transaction;
+
+/// The genesis block as defined by CoinNet
+public immutable Block GenesisBlock =
+{
+    header:
+    {
+        prev_block:  Hash.init,
+        height:      Height(0),
+        merkle_root: GenesisMerkleRoot,
+    },
+    txs: [ GenesisTransaction ],
+    merkle_tree: [ GenesisMerkleRoot ],
+};
```

Comment:
Since there is no Enrollment data on CoinNet, we need another way to succeed in the **System Integration Test**.

Reply:
It still uses the same data after the change. So yeah the Coinnet block needs to be modified, but it belong to a separate PR, this is just a refactoring.